alpenhorn/io/_default_asyncs.py

```diff
@@ -45,6 +45,18 @@ def pull_async(
         The request we're fulfilling.
     """
 
+    # Recheck the database to see if a file hasn't shown
+    # up somehow
+    if io.node.filecopy_state(req.file) == "Y":
+        log.info(
+            f"Cancelling pull request for {req.file.path}: "
```

Comment:
```suggestion
            f"Cancelling pull request for {req.file.path} "
```
Not sure if that colon was meant to be there